c7n/query.py

```diff
@@ -361,6 +361,7 @@ class QueryResourceManager(ResourceManager):
         get_retry((
             'ThrottlingException',
             'RequestLimitExceeded',
+            'LimitExceededException',
```

Comment:
I don't think we should be retrying on this error as this is typically a response from AWS that indicates that your account has reached a maximum limit of a certain resource, AWS's API response behavior is somewhat inconsistent on this front, as documented here: https://github.com/aws/aws-sdk-go/issues/1376

I'm not sure how exactly we can get around this other than defining custom retryable errors in the class definition of the resource itself to make sure we aren't retrying when we shouldn't be.

Reply:
Thanks for the link.  This explains why our read only kinesis policy fails with LimitExceededException.  I'll see if I could make a better fix.